scripts/vet.sh

```diff
@@ -70,6 +70,9 @@ not git grep "\(import \|^\s*\)\"google.golang.org/grpc/interop/grpc_testing" --
 # - Ensure that no trailing spaces are found.
 not git grep '[[:blank:]]$'
 
+# - Check for tabs in Markdown files
+git grep $'\t' -- '*.md' | fail_on_output || echo "No tabs found in Markdown files."
```

Comment:
shouldn't we be echoing only for failures?

Reply:
this might be enough `not git grep $'\t' -- '*.md'` because command and comment is self explanatory

Reply:
Done